biome.json

```diff
@@ -14,6 +14,7 @@
 		],
 		"ignore": [
 			"optuna_dashboard/ts/components/PlotlyColorTemplates.ts",
+			"tslib/react/src/components/PlotlyDarkMode.ts",
 			"standalone_app/src/PlotlyDarkMode.ts",
```

Comment:
Can we just remove this line?

Reply:
Removed L18 (`"standalone_app/src/PlotlyDarkMode.ts"`), not L17 (`"tslib/react/src/components/PlotlyDarkMode.ts"`), thank you for your catch!